Dockerfile

```diff
@@ -1,6 +1,10 @@
 # Use the latest version with Python 3.10
 FROM continuumio/miniconda3:23.3.1-0
 
+# Detect architecture
+ARG TARGETPLATFORM="linux/amd64"
```

Comment:
Not used? Also, Buildkit (which is default builder since docker 23.0) always sets TARGETPLATFORM. Maybe we can  remove the default value for clarity

Reply:
Good catch! Fixed